rosbridge_library/src/rosbridge_library/capabilities/call_service.py

```diff
@@ -35,12 +35,17 @@
 from rosbridge_library.capability import Capability
 from rosbridge_library.internal.services import ServiceCaller
 
+import sys
+if sys.version_info >= (3, 0):
+    string_types = str
+else:
+    string_types = (str, unicode)
```

Comment:
Can you move this into `rosbridge_library/utils/__init__.py` and import from there? Then, we can avoid copy and pasting of the same functionality.

Reply:
Yes, that seems better 👍